mars/services/task/supervisor/processor.py

```diff
@@ -50,6 +50,36 @@ async def inner(processor: "TaskProcessor", *args, **kwargs):
 
     return inner
 
+def format_graph_data(graph: TileableGraph):
```

Comment:
Maybe it is better to put this code block inside the method implementation, as it is not be used elsewhere.